src/js/gi/selectors/estimator.js

```diff
@@ -0,0 +1,203 @@
+import { createSelector } from 'reselect';
+
+const getConstants = (state) => state.constants.constants;
+
+const getEligibilityDetails = (state) => {
+  const details = Object.assign({}, state.eligibility);
+  delete details.dropdowns;
+  return details;
+};
+
+const getRequiredAttributes = (_state, props) => {
+  const { type, bah } = props;
+  return { type, bah };
+};
```

Comment:
If the two above `get*` functions are getting used in a few different modules, could they be in `utils/helpers.js` and imported to where they're needed?